<?xml version="1.0"?>
<launch>
  <arg name="chain_start" default="torso_lift_link" />
  <arg name="chain_end" default="r_wrist_roll_link" />
  <arg name="timeout" default="0.005" />

  <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find pr2_description)/robots/pr2.urdf.xacro'" />


  <node name="trac_ik" pkg="reachability" type="ik"  output="screen">
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="urdf_param" value="/robot_description"/>
  </node>


</launch>
